x-pack/test/cloud_security_posture_api/routes/vulnerabilities_dashboard.ts

```diff
@@ -186,16 +186,13 @@ export default function ({ getService }: FtrProviderContext) {
 
   describe('Vulnerability Dashboard API', async () => {
     beforeEach(async () => {
+      await index.removeFindings();
```

Comment:
I wonder why `afterEach` doesn't handle it. Do these tests depend on other tests somehow?